wordpress-mobile/WordPress-Android
# Contain and retain GB on rotation

Fixes state loss issues on Gutenberg-mobile ([example](https://github.com/wordpress-mobile/gutenberg-mobile/issues/377))

### Implementation details

The Gutenberg-mobile editor is of course living inside a React Native view (ReactRootView). The solution here manages the View via a retained (headless) Fragment (GutenbergContainerFragment.java).

Especially when rotating, `setContent()` is bypassed so the content is not written again. It's not needed and it will also cause problems in the GB state.

To test:

* Open a Gutenberg post in the editor
* Play around, making changes to the post if desired
* Rotate the device. Notice how fast the rotation performs and, notice the post changes still be there, along with the undo/redo history.

Note: at the time of writing, the Travis build fails, apparently due to some trouble with Jitpack checking out the branch (`Git error. Checkout conflict with file: Cannot delete file: react-native-aztec/example/ios`). Opened ticket on JitPack [here](https://github.com/jitpack/jitpack.io/issues/3707).

Update release notes:
